### PR TITLE
feat(fa): Adds employee email

### PR DESCRIPTION
**Pull request by Rikkiabb**
# ...
https://app.asana.com/0/1201427499183230/1201234405372298

## What

Sends email to employee when the employee is created

## Why

Better onboarding

## Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] Formatting passes locally with my changes
- [X] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/financial-aid/backend/src/environments/environment.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ const prodConfig = {
     },
   },
   baseUrl: process.env.BASE_URL,
+  veitaUrl: process.env.VEITA_BASE_URL,
```
Comment (MargretFinnboga) [inline apps/financial-aid/backend/src/environments/environment.ts:35]: baseUrl and veitaUrl, do we have better or more clear names 😄

File: apps/financial-aid/backend/src/app/modules/staff/staff.service.ts [hunk 2]
```diff
@@ -63,11 +69,43 @@ export class StaffService {
     return { numberOfAffectedRows, updatedStaff }
   }
 
+  private async sendEmail(
+    input: CreateStaffDto,
+    municipalityName: string,
+    user: Staff,
+  ) {
+    if (input.roles.includes(StaffRole.EMPLOYEE)) {
```
Comment (MargretFinnboga) [inline apps/financial-aid/backend/src/app/modules/staff/staff.service.ts:77]: Just employee? Should admin and super admin not get email? 📫

**Review by MargretFinnboga — APPROVED** [review]
few questions 🥑 ?

File: apps/financial-aid/backend/src/app/modules/staff/staff.service.ts (hunk 2 above)
Comment (Rikkiabb) [reply apps/financial-aid/backend/src/app/modules/staff/staff.service.ts:77]: Nopes they get another one, which is not designed yet

File: apps/financial-aid/backend/src/environments/environment.ts (hunk 1 above)
Comment (Rikkiabb) [reply apps/financial-aid/backend/src/environments/environment.ts:35]: Sure, any suggestions? 😄

**Review by MargretFinnboga — APPROVED** [review]
🌮 Olay

**Pull request merged**